app/controllers/api/groups_controller.rb

```diff
@@ -1,6 +1,12 @@
 class API::GroupsController < API::RestfulController
   load_and_authorize_resource only: :show, find_by: :key
   load_resource only: [:upload_photo, :use_gift_subscription], find_by: :key
+  skip_before_action :authenticate_user!, only: :index
```

Comment:
This was left in from when Rob and I were trying to make it work for logged out users. It doesn't work. I'll remove. However, when you try and access the explore page as a logged out user you get redirected to sign in page. Would be keen to work out how to avoid that. I think the BaseController ```  before_filter :authenticate_user!``` is being applied.